code/modules/research/techweb/_techweb.dm

```diff
@@ -48,6 +50,18 @@
 	id = "SCIENCE"
 	organization = "Nanotrasen"
 
+/datum/techweb/bepis	//Should contain only 1 BEPIS tech selected at random.
+	id = "SCIENCE"
+	organization = "Nanotrasen R&D"
+
+/datum/techweb/bepis/New()
+	. = ..()
+	var/bepis_id = pick(experimental_nodes)	//To add a new tech to the BEPIS, add the ID to this pick list.

```

Comment:
I fail to see the need for these to be separate from hidden nodes. They're functionally identical.